src/Pages/BookPage.js

```diff
@@ -0,0 +1,166 @@
+import React, { useState, useEffect } from 'react';
+import {
+  Card, CardMedia, CardContent, Typography, Paper, Link as LinkUI, List, ListItem, ListItemText,
+} from '@mui/material';
+import { useParams } from 'react-router-dom';
+
+const Airtable = require('airtable');
+
+/* This is the default image if TCK forgets to populate the image field. */
+const TCK_LOGO = 'https://s3.us-west-2.amazonaws.com/secure.notion-static.com/119db2be-192f-4a50-94ff-2c4f021384e3/TCK_PNG_Logo.png?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Content-Sha256=UNSIGNED-PAYLOAD&X-Amz-Credential=AKIAT73L2G45EIPT3X45%2F20220124%2Fus-west-2%2Fs3%2Faws4_request&X-Amz-Date=20220124T032111Z&X-Amz-Expires=86400&X-Amz-Signature=bf863cbd890a1ac3d8ef071ad1b02652642ae24de0e5f10c6fcc4bfcef62a97f&X-Amz-SignedHeaders=host&response-content-disposition=filename%20%3D%22TCK%2520PNG%2520Logo.png%22&x-id=GetObject';
+
+const base = new Airtable({ apiKey: process.env.REACT_APP_AIRTABLE_USER_KEY })
+  .base(process.env.REACT_APP_AIRTABLE_BASE_KEY);
+
+function BookPage() {
+  const [book, setBook] = useState();
+  const [author, setAuthor] = useState();
+  const [illustrator, setIllustrator] = useState();
+
+  // Instead of using props, we pull bookId from URL
+  const params = useParams();
+  const { bookId } = params;
+
+  useEffect(() => {
+    const getEntry = async (tableName, entryId, setter) => new Promise((resolve, reject) => {
+      base(tableName).find(entryId, (err, entryRecord) => {
+        if (err) {
+          reject();
+        }
+        setter(entryRecord);
+        resolve(entryRecord);
+      });
+    });
+
+    // This is similar to promise chaining with .then() calls,
+    // but in a (hopefully) more succinct way
+    const getEntries = async () => {
+      const bookRecord = await getEntry('Book', bookId, setBook);
+      const authorId = bookRecord.get('author');
+      const illustratorId = bookRecord.get('illustrator');
+      getEntry('Creator', authorId, setAuthor);
+
+      getEntry('Creator', illustratorId, setIllustrator);
+    };
+    getEntries();
+  }, [bookId]); // Runs on mount and on change of bookId
+
+  const getDefault = (x, defaultX) => {
+    if (x) {
+      return x;
+    }
+    return defaultX;
```

Comment:
Here, and throughout the code, you can make your conditionals more concise by using [ternary operators](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator)! 

If you make that replacement, the code becomes just 1 line long!
```suggestion
    return x ? x : defaultX;
```